openassessment/xblock/static/js/src/studio/oa_edit_validation_alert.js

```diff
@@ -86,6 +86,7 @@ OpenAssessment.ValidationAlert.prototype = {
 
             $('.oa_editor_content_wrapper', this.editorElement).each(function() {
                 $(this).css(styles);
+                $(this).scrollTop($(this).scrollTop() + alertHeight); //keep our relative scroll position the same
```

Comment:
@cahrens This fixes the weird scrolling behavior you were seeing. This was actually always happening, with every assessment type, but all the other types shrink in height when the validation alert is displayed so it never mattered.

The intent here is to tell the `editorElement` to set its scroll position in such a way as to not change what is displayed when the alert appears.